templates/navigation-mobile-menu-items.twig

```diff
@@ -0,0 +1,63 @@
+{% for key,item in menu %}
```

Comment:
Needed to move to a template since [this Sonar Cloud reported issue](https://github.com/greenpeace/planet4-master-theme/pull/1634/checks?check_run_id=5391112985)

```
Quality Gate failed

Failed condition B Reliability Rating on New Code (is worse than A)
Failed condition 35.3% 35.3% Duplicated Lines (%) on New Code (is greater than 3%)
```